rust/basic_dao/src/basic_dao/src/basic_dao.did

```diff
@@ -0,0 +1,126 @@
+type BasicDaoStableStorage = record {
```

Comment:
I would link to this file from the README so users can easily see the interface.